### ISSUE TITLE
Impairment of fructose metabolism

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4268]
As we consider how to refactor the term labels in the metabolism section of the HPO, here is a good term to think about:
https://hpo.jax.org/app/browse/term/HP:0011033

We have agreed that we should try to say "Abnormal level/concentration/amount" etc rather than "Abnormal homeostasis of X", but there are a number of terms in this section that do represent abnormal homeostasis items.

It seems that there is a similar split between morphology (in this case, levels/concentrations) and physiology (abnormal responses to ingestion of a substance, for instance) as we have on other areas of the HPO. 

Should we restructure the "metabolism" section in this way? Should we consider even changing the name of the top level to something that would be like "Abnormal laboratory measurement"?
@mellybelly @drseb @LCCarmody @cmungall @nicolevasilevsky @jpgourdine @jmcmurry

**Comment by jpgourdine** [comment]
@pnrobinson "Abnormal laboratory measurement" sounds great especially for the molecular phenotypes

**Comment by drseb** [comment]
I like the new label.

Not sure about the restructuring though

**Comment by pnrobinson** [comment]
@drseb -- sorry if I am being unclear, but this is the same thing I thought we agreed was good. Maybe we should set up a skype before we move forward to make sure we are on the same page?

**Comment by pnrobinson** [comment]
@drseb @mellybelly @cmungall others
How about something like this
Abnormality of metabolism/homeostasis (leave current top level label)
-Abnormal level of analyte
-----This would have things like reduced calcium level
-Abnormal metabolism
-----This would have "functional items" like fructose intolerance
-Presence of xenobiotic

There are a few other things like bloodstream infectious agents, but 95% of terms would fit well with the above 3 categories, I think. I will implement unless somebody shouts next week

**Comment by pnrobinson** [comment]
I have relabeled the ions. There is still work to be done with restructuring the metabolism section.

**Comment by pnrobinson** [comment]
@drseb @cmungall @matentzn @mellybelly 
I have finished a first re-structuring of the metabolism section.
Basically, this means that I relabeled to Abnormal concentration for most of the grouping terms. There are terms that generally refer to homeostasis or metabolism, and I put them now in the correct subhierarchy.
The Metabolism section is now structured largely according to what is truly observed. It was previously kind of a mess.
It probably still needs work, but I think it is much better. 
Now, we will need to revised the logical definitions. I am going to close this issue and I would recommend we open new issues to discuss the patterns if necessary.
